.github/workflows/scanner-versioned-definitions-update.yaml

```
@@ -82,7 +82,7 @@ jobs:
         echo "$code" | grep -q 200
 
     - name: Download NVD
-      if: github.event_name != 'pull_request'
+      if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'pr-update-scanner-nvd')
```

For some reason, my brain is having a hard time parsing this. What'd you think about factoring out the negation?

```suggestion
      if: !(github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'pr-update-scanner-nvd'))
```